src/ForceTorque.cc

```diff
@@ -106,6 +130,28 @@ Errors ForceTorque::Load(ElementPtr _sdf)
     }
   }
 
+  auto loadAxisNoise = [](sdf::ElementPtr _parent,
+                          const std::string _groupLabel,
+                          const std::string _axisLabel,
+                          sdf::Noise& _noise)
+  {
+    if (_parent->HasElement(_groupLabel) &&
+        _parent->GetElement(_groupLabel)->HasElement(_axisLabel))
+    {
+        auto axis = _parent->GetElement(_groupLabel)->GetElement(_axisLabel);
+        _noise.Load(axis->GetElement("noise"));
```

Comment:
Can you capture the errors from `sdf::Nosie::Errors` and copy them over to `errors` like so https://github.com/ignitionrobotics/sdformat/blob/3b4d1362ad267d5acb96fca4a91f8620ad7b867c/src/Model.cc#L179-L183?